### ISSUE TITLE
[a11y] Cannot select tree item without expanding/collapsing contents

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1604]
Due to the nature of how trees work in `a11y`, there is no way to select a tree item without expanding or collapsing the contents. To select an item to merge or synonymize requires all of the children to either be opened or hidden to bring attention to the object.

**Comment by maxxxxxdlp** [comment]
Begun fixing this by allowing to select a node without collapsing it by right-clicking
Alternatively, you can double click on it (to collapse and uncollapse it).
Will do a better fix in the future

**Comment by grantfitzsimmons** [comment]
Is it possible to disable the browser's context menu when right-clicking the node?

**Comment by maxxxxxdlp** [comment]
Fixed.

Tree node can be selected without collapsing it by holding the command or shift key when clicking.

**Comment by grantfitzsimmons** [comment]
Documented on the [Speciforum](https://discourse.specifysoftware.org/t/trees-in-specify-7/534) for future reference
